drivers/wifi/siwx91x/siwx91x_wifi.c

```diff
@@ -20,7 +21,7 @@
 #include "sl_wifi.h"
 #include "sl_net.h"
 
-#define SIWX91X_INTERFACE_MASK (0x03)
+#define SIWX91X_DRIVER_VERSION "2025.6.0-alpha"
```

Comment:
I believe the version of this driver is actually "Zephyr 4.2". If you want to get the version of Wiseconnect SDK, you probably need to import a symbol from Wiseconnect.